x-pack/legacy/plugins/task_manager/lib/take_with_backpressure.test.ts

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
sadly, not familiar enough with rx to being able to review these bits (if I had some time, I'd use this as a great excuse to dig in, but I don't ATM).  Let's make sure an rx-savvy engineer reviews these bits.

Reply:
Fair enough, I'll see who I can find. :)